### DISCUSSION TITLE
能否支持以下几个需求啊

### DISCUSSION THREAD

**Discussion by hsyan2008** [rbatis/rbatis#273] [General]
1、字段加反引号，类似unique这样的字段报错了
2、指定查询的字段，有时候只要知道某些记录是否存在，却把所有字段都查出来了
3、tinyint，如果宽度设置为1，就必须为bool类型了，设置为u8就报错了，有时候只是用来状态的枚举，并不是is_XXX，可以增加个判断是否is开头
4、save_bach*能否增加一个支持wrapper的

**Comment by zhuxiujia** [comment]
> 1、字段加反引号，类似unique这样的字段报错了 2、指定查询的字段，有时候只要知道某些记录是否存在，却把所有字段都查出来了 3、tinyint，如果宽度设置为1，就必须为bool类型了，设置为u8就报错了，有时候只是用来状态的枚举，并不是is_XXX，可以增加个判断是否is开头 4、save_bach*能否增加一个支持wrapper的

问题1-可以考虑增加
问题2-需要思考下合理性
问题3-不知道是不是bug，也可能是依赖sqlx的驱动的bug，待确认
问题4-可以

**Comment by strive1216** [comment]
tinyint 这个字段类型(必须指定bool类型) 可以使用u8类型吗? 使用rust重构项目卡这了

**Comment by hsyan2008** [comment]
> tinyint 这个字段类型(必须指定bool类型) 可以使用u8类型吗? 使用rust重构项目卡这了

数据表里，那个长度设置为1的话，就必须是bool类型，如果你要用u8，就改成1以上就可以了，目前是这样的

**Comment by LeonRust** [comment]
> > tinyint 这个字段类型(必须指定bool类型) 可以使用u8类型吗? 使用rust重构项目卡这了
> 
> 数据表里，那个长度设置为1的话，就必须是bool类型，如果你要用u8，就改成1以上就可以了，目前是这样的

这个问题比较诡异，我遇到了下面的情况：

本地环境： Mysql8.0.28 字段类型为tinyint rust设置为u8是OK的。
服务器生产环境：Mysql8.0.23 字段类型为tinyint rust设置为u8是则报错，提示必须为bool类型。

在Mysql5中，tinyint(1)必须为bool类型。

**Comment by hsyan2008** [comment]
> > 1、字段加反引号，类似unique这样的字段报错了 2、指定查询的字段，有时候只要知道某些记录是否存在，却把所有字段都查出来了 3、tinyint，如果宽度设置为1，就必须为bool类型了，设置为u8就报错了，有时候只是用来状态的枚举，并不是is_XXX，可以增加个判断是否is开头 4、save_bach*能否增加一个支持wrapper的
> 
> 问题1-可以考虑增加 问题2-需要思考下合理性 问题3-不知道是不是bug，也可能是依赖sqlx的驱动的bug，待确认 问题4-可以

预计什么时候可以有啊
